src/types/h3.ts

```diff
@@ -0,0 +1,19 @@
+import type { NitroFetchRequest, $Fetch } from "./fetch";
+
+export type H3EventFetch = (
+  request: NitroFetchRequest,
+  init?: RequestInit
+) => Promise<Response>;
+
+export type H3Event$Fetch = $Fetch<unknown, NitroFetchRequest>;
+
+declare module "h3" {
+  interface H3Event {
+    /** @experimental Calls fetch with same context and request headers */
+    fetch: H3EventFetch;
+    /** @experimental Calls fetch with same context and request headers */
+    $fetch: H3Event$Fetch;
+  }
+}
+
+export {};
```

Comment:
Locating these types here will not result in augmenting `H3Event` in a Nuxt/Nitro project

Reply:
Testing build archive against nitro-deploys, seems to be working fine:

<img width="694" alt="image" src="https://github.com/unjs/nitro/assets/5158436/3a84ff00-5bd1-4f90-9218-70e9439231ca">


<img width="952" alt="image" src="https://github.com/unjs/nitro/assets/5158436/c0938ff4-78f9-4e76-a7d5-68e13fb361ca">